Automattic/wp-calypso
# Store: Update/product categories data layer

This commit changes the code from doing requests in an action thunk to
using the data-layer http mechanism.

To Test:
1. Run `npm test`
2. Run `npm start`
3. Visit the product create page and update pages and verify that the categories appear in the drop-down as they did before.
